### ISSUE TITLE
Custom grid_search displaying

### ISSUE THREAD

**Issue by OuterAlt** [APY/APYDataGridBundle#922]
Hi,

I wanted to customize the search filters (who are displayed with `{{ grid_search(grid) }}`.
So i can see all my filters with the Search and reset buttons, but now i want to customize them, but i didn't manage to.

Would you tell me how to do so ?

Best regards,
Thomas

**Comment by DonCallisto** [comment]
Hi Thomas, your question seems no clear at all to me.
Please, try to reword it again an be more precise.

However I'm just new to this bundle, it's not guaranteed I can find a solution.

**Comment by OuterAlt** [comment]
Hello @DonCallisto, 

Thanks for answering.
I'm very new to it too, but i've found my solution.

I'll re-explain it too though :)
-- I managed to display correctly the Grid
-- I managed to display the filters with `{{ grid_search(grid) }}`

The problem was that i wanted to customize them with CSS, but i wasn't successful because i haven't override correctly the template, thing that i've fixed. And now, everything is ok.

Thomas
